src/hyp3_isce2/burst.py

```diff
@@ -337,3 +338,24 @@ def get_product_name(
         The name of the interferogram product.
     """
     return f'{reference_scene}x{secondary_scene}'
+
+
+def search_cmr_uat(scene_name: str) -> asf_search.ASFSearchResults:
```

Comment:
This can be a TODO for the future, but in the period of time when CMR is NOT back-populated it would be great to check the date of the burst up-front and raise an error if it's date is before burst ingesting began. Something like:

`ValueError('Bursts before [date] are not currently available from ASF')`

Reply:
Stuck this in an issue: https://github.com/ASFHyP3/hyp3-isce2/issues/84